test/unit-tests/function/trigonometry/sinh.test.js

```diff
@@ -73,7 +73,7 @@ describe('sinh', function () {
   it('should return the sinh of a complex number', function () {
     approx.deepEqual(sinh(complex('1')), complex(1.1752011936438014, 0), EPSILON)
     approx.deepEqual(sinh(complex('i')), complex(0, 0.8414709848079), EPSILON)
-    approx.deepEqual(sinh(complex('2 + i')), complex(1.9596010414216, 3.1657785132162), EPSILON)
+    approx.deepEqual(sinh(complex('2 + i')), complex(1.9596010414216, 3.1657785132162), 1e-13)
   })
```

Comment:
This test file should not have one EPSILON that it uses throughout except for several occurrences of a _different_ hardcoded number. I take it you changed these occurrences because they weren't passing otherwise? Do you know if the values listed in the test are closer to the truth or the values math.js is producing are closer to the truth? If the latter, it seems you should correct the values listed in the test and go back to EPSILON. If the former, then please define COMPLEX_EPSILON at the top, maybe comment there why a separate accuracy is needed, and then use COMPLEX_EPSILON instead of these hard-coded numbers which could get lost in the shuffle. Also then it might be nice to open an issue that sinh for complex numbers is not quite as precise as sinh for reals is, if you have a moment to do so. Thanks!

Reply:
Oh that's the reference results are not closer to the truth enough, I improved the reference results by providing more precise bits. No more hard-coded numbers.